plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -328,31 +335,78 @@ Events emitted on deposit:
         address indexed token,
         uint256 amount
     );
+
+    event EthWithdrawn(
+        address indexed receiver,
+        uint256  amount
+    );
 ```
 
-Events emitted on withdrawal:
+For all events of the vault, see contract documentation: [ETH vault doc](https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/docs/contracts/EthVault.md#contract-members)
+
+
+### ERC20 Vault
 ```
+    event DepositCreated(
+        address indexed depositor,
+        uint256 indexed blknum,
+        address indexed token,
+        uint256 amount
+    );
+
     event Erc20Withdrawn(
         address payable indexed receiver,
         address indexed token,
         uint256 amount
     );
 ```
 
+For all events of the vault, see contract documentation: [ERC20 vault doc](https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/docs/contracts/Erc20Vault.md#erc20vaultsol)
 
 # Exit Game
-Exit Games handle all the actions around exits, challenges, etc.
+Exit Games handle all the actions around exits, challenges, etc. Each transaction type has one and only one exit game contract that provides the means of exiting funds from a transaction of that type.
+
+Once a exit is canonical to exit after the challenge period, it can be processed according to the exit priority and withdraw the fund back from Child chain to Root chain.
```

Comment:
```suggestion
Once an exit is canonical to exit after the challenge period, it can be processed according to the exit priority and withdraws the funds back to the Root chain from the Child chain.
```